src/main/scala/org/broadinstitute/dsde/workbench/avram/dataaccess/HttpSamDao.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.workbench.avram.dataaccess
+
+import java.util.logging.Logger
+
+import com.softwaremill.sttp._
+import io.circe.Decoder.Result
+import io.circe.{DecodingFailure, Json, ParsingFailure}
+import io.circe.generic.auto._
+import io.circe.parser._
+import org.broadinstitute.dsde.workbench.avram.util.{ErrorResponse, RestClient}
+
+class HttpSamDao extends SamDao with RestClient {
+  private val log: Logger = Logger.getLogger(getClass.getName)
```

Comment:
It seems `RestClient` has a `Logger`, could you make it `protected` and use that one?

Also could consider mixing in a trait like `LazyLogging` (see https://github.com/lightbend/scala-logging).

Reply:
Might be able to use `scala-logging`. Ultimately for App Engine, it has to boil down to `java.util.logging`. `getClass.getName` is going to return the name of the class that owns the logger so I think the logger needs to be in the immediate class (at least with `java.util.logging`).